packages/edit-navigation/src/utils/index.js

```diff
@@ -5,8 +5,8 @@ import { normalizePath } from '@wordpress/url';
 
 /**
  * The purpose of this function is to create a middleware that is responsible for preloading menu-related data.
- * It uses data that is returned from the /__experimental/menus endpoint for requests
- * to the /__experimental/menu/<menuId> endpoint, because the data is the same.
+ * It uses data that is returned from the /wp/v2/menus endpoint for requests
+ * to the /wp/v2/menu/<menuId> endpoint, because the data is the same.
```

Comment:
I think this is the last one. It was hiding in regex 😄 

https://github.com/WordPress/gutenberg/blob/d89d485100cc0bf9d116582ff642a1b7059cc0e5/packages/edit-navigation/src/utils/index.js#L47

Reply:
Fixes in c12aa8f

Reply:
Good catch, thank you 😄